app/scripts/shell/dimSearchFilter.directive.js

```diff
@@ -259,6 +259,9 @@
         } else if (term.indexOf('quality:') >= 0 || term.indexOf('percentage:') >= 0) {
           filter = term.replace('quality:', '').replace('percentage:', '');
           addPredicate("quality", filter);
+        } else if (term.indexOf('stat:') >= 0) {
+          filter = term.split(':')[1];
+          addPredicate(filter, term.split(':')[2]);
```

Comment:
Should you check to make sure that 2nd element is defined? If you type "stat:aimassist", will it pass undefined as the value for the filter? I think the stat filter expects whatever value it receives to have a length property and undefined doesn't.